app/src/main/java/com/nadosunbae_andorid/presentation/ui/review/ReviewDetailActivity.kt

```diff
@@ -0,0 +1,77 @@
+package com.nadosunbae_andorid.presentation.ui.review
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import androidx.activity.viewModels
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.bumptech.glide.Glide
+import com.nadosunbae_andorid.R
+import com.nadosunbae_andorid.data.model.response.review.ReviewTagBoxData
+import com.nadosunbae_andorid.databinding.ActivityReviewDetailBinding
+import com.nadosunbae_andorid.presentation.base.BaseActivity
+import com.nadosunbae_andorid.presentation.ui.review.adapter.ReviewTagBoxAdapter
+import com.nadosunbae_andorid.presentation.ui.review.viewmodel.ReviewDetailViewModel
+import com.nadosunbae_andorid.presentation.ui.review.viewmodel.ReviewListViewModel
+
+class ReviewDetailActivity : BaseActivity<ActivityReviewDetailBinding>(R.layout.activity_review_detail) {
+
+    private lateinit var reviewTagBoxAdapter: ReviewTagBoxAdapter
+
+    private val reviewDetailViewModel: ReviewDetailViewModel by viewModels {
+        object : ViewModelProvider.Factory {
+            override fun <T : ViewModel> create(modelClass: Class<T>): T {
+                return ReviewDetailViewModel() as T
+            }
+        }
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        initTagBoxAdapter()
+        setClickListener()
+        observeBackground()
+        setTestData()
+
+    }
+
+    private fun initTagBoxAdapter() {
+        reviewTagBoxAdapter = ReviewTagBoxAdapter()
+        binding.rvReviewDetail.adapter = reviewTagBoxAdapter
+    }
+
+    private fun setClickListener() {
+        binding.btnBack.setOnClickListener {
+            finish()
+        }
+    }
+
+    private fun observeBackground() {
+        reviewDetailViewModel.urlBackground.observe(this, Observer {
```

Comment:
```suggestion
        reviewDetailViewModel.urlBackground.observe(this){
        }
```
지금 형기 선배가 쓴 부분은 다빈이가 그때 말해줬을 때 자바식이라고 했던 것 같아! 그래서 위와 같이 간단하게 쓸 수 있다고 하더라고 참고하시면 좋겠습니당

Reply:
오호 감사선배! 수정하겠습니닷